steps/zip-directory-and-export-its-path/v1.0.0/step.yml

```diff
@@ -0,0 +1,57 @@
+title: Zip Directory and Export its Path
+summary: |
+  This step will zip a directory and then export the path as an environment variable
+description: |
+  This step will zip a directory and then export the path as an environment variable
+website: https://github.com/danielmcconville/bitrise-step-zip-directory-and-export-its-path
+source_code_url: https://github.com/danielmcconville/bitrise-step-zip-directory-and-export-its-path
+support_url: https://github.com/danielmcconville/bitrise-step-zip-directory-and-export-its-path/issues
+published_at: 2018-11-16T10:19:16.468536521+10:30
+source:
+  git: https://github.com/danielmcconville/bitrise-step-zip-directory-and-export-its-path.git
+  commit: d975d583a6a03345047c98a3feba6606b41fddeb
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```

Comment:
Neither git nor wget seems to be needed here. The only dependency is a `zip`.

Reply:
Good point. Sorry I missed that, this is the first time I have had a go at uploading a step. That was just the defaults from the template.

I have pushed an update.